lid_utils.py

```diff
@@ -22,7 +24,7 @@ def predict(self, text):
         language_code, script = label.split("_")
         assert len(language_code) == 3
 
-        return f"[{language_code}:{logit[0]}]"
+        return language_code, logit[0]
```

Comment:
@AMR-KELEG I realize you might not be completely onboard with this, so I would be very happy for your feedback (and also quite happy to ditch the PR altogether if you do not like it :))

Reply:
@mrshu I am fine with this change, especially if it will help you build the ensembling model.
It will break some scripts, but it is still easy to fix them, so I am happy to do so if you want to merge this PR.

Reply:
@AMR-KELEG I'd be very happy to fix those scripts -- I really do not think they should be parsing strings on their own, as that will probably duplicate effort, which I believe it would be best to avoid at this point :)